src/exec-dmenu.js

```diff
@@ -3,9 +3,12 @@ const { exec } = require('child_process')
 module.exports = (...args) => choices =>
   new Promise((resolve, reject) => {
     let choice = ''
+    let dmenu_path = process.env.DMENU_PATH
```

Comment:
lets push all the logic into this assignment, and move it above the function declaration. It does not need to be assigned every time the function is called.

a combined version of this might look like
```javascript
const dmenu_path = process.env.DMENU_PATH || 'dmenu'
```

Reply:
Thanks, I didn't actually know about `||` since I don't know JS. Useful. :)

Reply:
no problem! You just have to delete this line and we're all set.